docs/cli-client/gov/submit-proposal.md

```diff
@@ -30,14 +30,13 @@ iriscli gov submit-proposal --help
 | --version           |            0                | [uint64] the version of the new protocol                                                                            |       |
 | --software           |           " "                 | [string] the software of the new protocol                                                                         |       |
 | --switch-height           |       0                     | [string] the switchheight of the new protocol                                                         |       |
-| --threshold | "0.85"   |  [string] the upgrade signal threshold of the software upgrade                                                   |               |
 
 ## Examples
 
 ### Submit a 'ParameterChange' type proposal
 
 ```shell
-iriscli gov submit-proposal --chain-id=test --title="update MinDeposit proposal" --param='mint/Inflation=0.050' --type=ParameterChange --description="a new parameter change proposal" --from=node0 --fee=0.01iris
+iriscli gov submit-proposal --chain-id=test --title="update MinDeposit proposal" --param='mint/Inflation=0.050' --type=ParameterChange --description="a new parameter change proposal" --from=node0 --fee=0.01iris --threshold=0.9
 ```
```

Comment:
There is no `threshold` for ParameterChange proposal. Please remove it.

Reply:
fixed